### ISSUE TITLE
Wrong data for Brasilia-Brazil

### ISSUE THREAD

**Issue by l3x0** [chubin/wttr.in#626]
https://wttr.in and https://wttr.in/brasilia
are consistently showing data above the maximum historically recorded temperature 36.7 C
https://v2.wttr.in/brasilia
https://v3.wttr.in/brasilia
are showing data for Mato Grosso, a nearby state. Brasilia is Brazil's capital, located in the Federal District (DF)
The location for https://v4.wttr.in/brasilia
[-10.3333332,-53.1999999]
Is in the middle of Mato Grosso state
![Screenshot from 2021-09-17 08-58-04](https://user-images.githubusercontent.com/83312283/133779286-52f45365-401f-47ac-aa28-2b4df5551c9c.png)

**Comment by chubin** [comment]
Agree. Something is wrong here. I need to investigate it.
Could you please check if this location https://wttr.in/Guara is resolved correctly?
Are the coordinates in the Location: line correct?

**Comment by l3x0** [comment]
Yes, https://wttr.in/Guara checks out for Guara city, Sao Paulo state (SP), let me know if I can help in any other way

**Comment by chubin** [comment]
I believe I've fixed it. Please test. Now though, we have a different problem (#400), but it existed already.

**Comment by l3x0** [comment]
https://wttr.in looks good for Brasilia, Federal District, Brazil here. 'v2' and 'v3' are still linking to Cuiaba in Mato Grosso State.

**Comment by chubin** [comment]
Good catch! I will fix it for v2/v3 too.
